src/components/views/rooms/MessageComposer.js

```diff
@@ -42,6 +42,17 @@ const formatButtonList = [
     _td("numbered-list"),
 ];
 
+function Avatar(props) {
```

Comment:
Can we call this `ComposerAvatar` or something to avoid potential confusion with the other avatar components?